Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -561,6 +575,8 @@ void MediaPlayerPrivateGStreamer::seekToTarget(const SeekTarget& inTarget)
         }
     }
 
+    ASSERT(!m_isSeeking);
+    m_wasPausedBeforeSeeking = paused();
```

Comment:
The field is named `m_wasPausedBeforeSeeking`, but if we arrive from the `// We can seek now.` we're setting this value *after* the seek, not before.

Reply:
Well, I don't follow all seeka shananigans but if here we are setting `m_isSeeking = true`, we need to set `m_wasPausedBeforeSeeking` before that.